app/components/restructured_work_history/job_component.html.erb

```diff
@@ -14,6 +14,14 @@
 
     <% if @work_experience.relevant_skills %>
       <p class="govuk-body-s govuk-!-margin-bottom-1">This role used skills relevant to teaching</p>
+    <% elsif @work_experience.relevant_skills.nil? && @editable %>
+      <%= govuk_inset_text(classes: 'govuk-error-summary__list app-inset-text--error') do %>
```

Comment:
We should probably avoid mixing component classes (summary list + inset text) here.

Could you add this to the the `/app/frontend/styles/_inset-text.scss` file to make the links red and bold:

```scss
.app-inset-text--error {
  .govuk-link {
    @include govuk-typography-weight-bold;
    color: $govuk-error-colour;
  }
}
```

And then you can drop the extra class from the inset text:

```suggestion
      <%= govuk_inset_text(classes: 'app-inset-text--error') do %>
```